tests/functional/l/logging_not_lazy.py

```diff
@@ -14,7 +14,6 @@
 renamed_logging.warn('%s' + ' the rest of a single string') # [logging-not-lazy]
 renamed_logging.log(renamed_logging.INFO, var_name + var) # [logging-not-lazy]
 
-var_name = 'Var:'
```

Comment:
Why remove this line ?

Reply:
I didn't think it added anything after line 8. Seems a duplicate